fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/impl/FHIRPersistenceJDBCImpl.java

```diff
@@ -473,18 +516,23 @@ public <T extends Resource> SingleResourceResult<T> create(FHIRPersistenceContex
      * @param requestShard
      * @param searchParameters
      */
-    private void sendParametersToRemoteIndexService(String resourceType, String logicalId, long logicalResourceId, 
+    private void storeSearchParameterValues(String resourceType, String logicalId, long logicalResourceId, 
             int versionId, java.time.Instant lastUpdated, String requestShard, 
-            ExtractedSearchParameters searchParameters) throws FHIRPersistenceException {
+            ExtractedSearchParameters searchParameters, String currentParameterHash) throws FHIRPersistenceException {
```

Comment:
I missed this one before, but javadoc could use an update for the new `currentParameterHash` param.
Also wondering if the description should be updated to give equal weight to the local case.
Something like this?
> Convert the extracted parameters into a SearchParametersTransport and either store it locally (to commit with the local transaction) or send it off to a remote indexing service (if so configured).

Reply:
ok, also added more detail to the parameter javadoc